saleor/payment/gateways/razorpay/forms.py

```diff
@@ -0,0 +1,69 @@
+from django import forms
+from django.forms.utils import flatatt
+from django.forms.widgets import HiddenInput
+from django.utils.html import format_html
+from django.utils.translation import pgettext_lazy
+
+from ...forms import PaymentForm
+
+CHECKOUT_SCRIPT_URL = 'https://checkout.razorpay.com/v1/checkout.js'
+
+PRIMARY_BTN_TEXT = pgettext_lazy(
+    'Rozorpay payment gateway primary button', 'Pay now with Razorpay')
+SECONDARY_TITLE = pgettext_lazy(
+    'Rozorpay payment gateway secondary title', 'Total payment')
+
+
+class RazorPayCheckoutWidget(HiddenInput):
+    def __init__(
+            self,
+            *, payment,
```

Comment:
We should pick some convention and stick to it, sometimes we put one argument per line, sometimes multiple ones. I'd go for one arg per line + type annotations whenever possible.

Reply:
The logic here was
1. Passed parameters;
2. Configured parameters;
3. Any Django/ other parameters.